orion/sites/admin/src/programs/programEdit.js

```diff
@@ -178,9 +178,43 @@ export class ProgramEditPage extends React.Component {
 }
 
 function checkProgram(program) {
-  // *todo* change this later!
+  let checkpro=true;
+
+  let programId = program.programId
+
+  //checks if programId contains special characters
```

Comment:
For comments, can you capitalize at the beginning and have a space after the `//`?
```
// Checks if ...
```